modules/virt-about-workload-security.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-workload-security_{context}"]
+= About workload security
+
+By default, virtual machine (VM) workloads do not run with root privileges in {VirtProductName}.
+
+For each VM, a `virt-launcher` pod runs an instance of `libvirt` to manage the VM process. The `virt-launcher` pod runs in _session mode_, meaning that all users in the `virt-launcher` container must have a nonzero user identifier (UID). Therefore, VMs run as unprivileged pods, adhering to the security principle of least privilege.
+
+[id="features-that-require-root-access_virt-about-workload-security"]
```

Comment:
Small suggestion: I would remove this header because it looks like a red flag to the user --- "OMG, which features require root? I better read that first!" What follows could, I think, be made more compact:

"Features that require root privileges, for example, Virtiofs (link?), are not supported." 

Since we are unlikely to maintain an ever-growing list of unsupported features,  I think it's safe to mention this as an example, rather than calling it out as a def list. :-)

Reply:
Good call! Thank you. I updated it so that it's in a note instead of a whole section. Let me know if you have any further comments. :)

Reply:
Honestly, I'm not sure about putting that in a note. If you just told the user that features requiring root privilege are not supported, then not supporting virtiofs is the default, not the exception to the rule. That's why I would use that as an example, rather than calling it out. OTOH, if it is an extremely important feature that users would expect to use, then it would justify a note.

Reply:
@xpivarc  Is the lack of support for virtiofs important enough to require drawing attention to it? In other words, would most users expect to be able to use it, so we should warn them that they can't? Or should we just mention that it is not supported?

Reply:
We don't support this feature at this moment. @vladikr Are you aware of any user despite our lack of support?

Reply:
I'm only aware of upstream KubeVirt users who explicitly know that this feature is experimental. virtiofs has never been productized as part of Openshift Virtualization.